src/Uno.Toolkit.UI/Controls/NavigationBar/NativeNavigationBarPresenter.iOS.cs

```diff
@@ -77,6 +79,12 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 				_navigationBar = navBar?.ResetRenderer(RendererFactory).Native;
 			}
 
+			if (navBar is { })
+			{
+				navBar.MainCommand.Click += OnMainCommandClick;
+				_mainCommandClickHandler.Disposable = Disposable.Create(() => navBar.MainCommand.Click -= OnMainCommandClick);
```

Comment:
same here.

Reply:
wait, setting the Disposable to a new Disposable doesn't dispose the previous one?!?